plugin-server/src/main/ingestion-queues/session-recording/blob-ingester/session-manager.ts

```diff
@@ -83,6 +56,33 @@ export class SessionManager {
         // this.lastProcessedOffset = redis.get(`session-recording-last-offset-${this.sessionId}`) || 0
     }
 
+    private async deleteFile(file: string, context: string) {
+        try {
+            await unlink(file)
+            status.info('🗑️', `blob_ingester_session_manager deleted file ${context}`, { file, context })
+        } catch (err) {
+            if (err && err.code === 'ENOENT') {
+                status.warn(
+                    '🤷‍♀️',
+                    `blob_ingester_session_manager failed deleting file ${context} path: ${file}, file not found. That's probably fine 🤷‍♀️`,
+                    {
+                        err,
+                        file,
+                        context,
+                    }
+                )
+                return
+            }
+            status.error('🧨', `blob_ingester_session_manager failed deleting file ${context}path: ${file}`, {
+                err,
+                file,
+                context,
+            })
+            captureException(err)
```

Comment:
note: I'm missing the team_id tag here still  ... but I'll just put up a follow-up PR for that